test/unit/mocked_data/test_get_bgp_neighbors/use-send-state-pr209/get-bgp-neighbor-information.xml

```diff
@@ -0,0 +1,422 @@
+<bgp-information>
+    <bgp-peer junos:style="detail">
```

Comment:
To make sure this will pass the tests, we must avoid the junos namespace, so please remove the `junos:` from everywhere.